src/fluree/db/json_ld/reify.cljc

```diff
@@ -339,10 +339,10 @@
                                            {:status 500 :error :db/invalid-commit})))
           assert         (db-assert db-data)
           retract        (db-retract db-data)
-          retract-flakes (retract-flakes db retract t-new iris)
+          retract-flakes (<? (retract-flakes db retract t-new iris))
           {:keys [flakes pid sid]} (<? (assert-flakes db assert t-new iris refs))
           all-flakes     (-> (empty (get-in db [:novelty :spot]))
-                             (into (<? retract-flakes))
+                             (into retract-flakes)
```

Comment:
I know the `<?` for retracts looked misplaced as it was, but it was intentional.

Its purpose was to run the fairly costly `assert-flakes` and `retract-flakes` in parallel on different threads, as opposed to serially.

This change makes it run serially.

Reply:
I made the original parallelization change here (in https://github.com/fluree/db/pull/349/commits/64a7fd7e275e377289c5e78a4e3b0cf33e1f5371) but recently noticed that it may not be safe b/c it's being passed a volatile map that is also being passed to assert-flakes.

So I reverted it back to the way it was before https://github.com/fluree/db/pull/349/commits/64a7fd7e275e377289c5e78a4e3b0cf33e1f5371.